src/DashboardService.Models.Db/DbComment.cs

```diff
@@ -0,0 +1,47 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Db;
+
+public class DbComment
+{
+  public const string TableName = "Comments";
+
+  public Guid Id { get; set; }
+  public Guid TaskId { get; set; }
+  public Guid CreatedBy { get; set; }
+  public string Content { get; set; }
```

Comment:
May replace it with CreatedBy, like in DbBoard.cs?

Reply:
++